Library/Homebrew/utils/ruby.sh

```diff
@@ -37,68 +75,27 @@ If there's no Homebrew Portable Ruby available for your processor:
 
   unset HOMEBREW_RUBY_PATH
 
-  if [[ "$HOMEBREW_COMMAND" == "vendor-install" ]]
```

Comment:
I think this form is more explicit and easier to follow for people less familiar with Bash.